pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x,z):

```

Comment:
should this perhaps be x and y as input variables?

Reply:
Sorry,  I've corrected that now, should pass the tests